Dockerfile

```diff
@@ -4,11 +4,10 @@ LABEL author="stakater"
 ARG SRC_DIR="ingress-monitor-controller"
 ENV SRC_DIR=${SRC_DIR}
 
+ADD bootstrap.sh /bootstrap.sh
+
+RUN chmod +x /bootstrap.sh
```

Comment:
No need to run this here, run this locally on the script so the file is committed with executable permissions

Reply:
done